sdk/sdk-api/src/main/java/mil/emp3/api/interfaces/IEvent.java

```diff
@@ -26,4 +26,16 @@
      * @return An object derived from {@link org.cmapi.primitives.IGeoBase}
      */
     public TargetType getTarget();
+
+    /**
+     * Sets a user object to let user track own events
+     */
+
+    public void setUserObject(Object userContext);
```

Comment:
We might want to investigate if we should start using @NonNull and @Nullable annotations in our code.